index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width">
+        <meta name="author" content="Amos Gloria">
+        <title>Facebook Clone | Welcome</title>
+        <link rel="stylesheet" href="./css/style.css">
+        </head>
+        <body>
+            <header>
+                <div class="header-block">
+                    <div id="branding">
+                        <h1><span class="highlight">facebook</span></h1>
+                        </div>
+                        <nav>
+                            <div class="facebool-lpgin">
+                                <div class="email">
+                                <label>Email or Phone</label><br>
+                                <input type="email or phone" placeholder="Email"><br>
+                                <label>Keep me logged in</label>
+                                </div>
+                                <div class="password">
+                                   <label>Password</label><br>
+                                   <input type="password" placeholder="Password"><br>
+                                   <a href="#">Forgotten your password?</a> 
+                                </div>
+                                <div class="login-button">
+                                    <input type="submit" value="Login">
```

Comment:
Try to ensure the login button is same height with `Password` and `Email` textboxes as it looks reduced a little.